server.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+type ToDoServer struct {
+	ToDoStore *ToDoStore
+	Count     int // total number of tasks (used as ID for individual task)
+}
+
+func NewServer() *ToDoServer {
+	return &ToDoServer{
+		ToDoStore: NewToDoStore(),
+	}
+}
+
+func (server *ToDoServer) GetToDos(w http.ResponseWriter, req *http.Request) {
+	for _, task := range server.ToDoStore.Tasks {
+		fmt.Fprintf(w, "ID: %d\nTask: %s\nCompleted: %t\n", task.ID, task.Description, task.Completed)
```

Comment:
Future improvement here would be to support multiple [content-types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type), if none is specified, or if `text/plain` is specified, then this is a good response. But if `application/json` is requested, then return a json array containing each task.

Reply:
noted. updated to return json array for now